src/theme/DocItem/Feedback.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import { FeedbackWidget, FeedbackWidgetText } from "@govtechsg/tradetrust-ui-components";
+import { gaEvent } from "@govtechsg/tradetrust-utils/dist/esm/analytics";
+
```

Comment:
docusaurus has its own react + webpack configuration, will have to patch webpack, ts load, install deps just to pass build etc, [tried](https://github.com/TradeTrust/documentation/pull/60/commits/6e06b8e4a6c5756b79e7cc6262ff1ca41b297d6f#diff-a038096cbdea434999e1dce5ab497212f1fe18204dde1a027ce3bdd663261a2aR13). 

maybe should release google analytics related on a new separate repo away from tradetrust-utils, to avoid deps hell from `fragments` related code in there. (somehow affecting)

hence, import js directly.